docs/developer-guide/base-maps/using-with-maplibre.md

```diff
@@ -0,0 +1,246 @@
+# Using with Maplibre
```

Comment:
Small nit, we always spell MapLibre with a capital L. It's a trademark we've registered, so we try to be consistent.